src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -50,30 +50,71 @@
  * </li>
  * </ul>
  * <p>
- * To prevent {@code FooCheck} violations from being reported write:
+ * To configure the default check
  * </p>
  * <pre>
- * &#64;SuppressWarnings("foo") interface I { }
- * &#64;SuppressWarnings("foo") enum E { }
- * &#64;SuppressWarnings("foo") InputSuppressWarningsFilter() { }
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;MemberName&quot;/&gt;
+ *   &lt;module name=&quot;ConstantName&quot;/&gt;
+ *   &lt;module name=&quot;ParameterNumber&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * Class Test {
+ *   &#64;SuppressWarnings({"membername"}) private int J;  // ok
+ *   &#64;SuppressWarnings({"constantname"}) private static final int m = 0;  // ok
+ *
+ *   &#64;SuppressWarnings("paramnum")
+ *   public void needsLotsOfParameters1(@SuppressWarnings("unused") int a,  // violation
+ *     int b, int c, int d, int e, int f, int g, int h) {
+ *     // ...
+ *   }
+ * }
  * </pre>
  * <p>
  * Some real check examples:
  * </p>
  * <p>
- * This will prevent from invocation of the MemberNameCheck:
+ * This will prevent selected violations from MemberNameCheck:
  * </p>
  * <pre>
- * &#64;SuppressWarnings({"membername"})
- * private int J;
+ * &lt;module name=&quot;TreeWalker&quot;&gt;
+ *   &lt;module name=&quot;MemberName&quot;/&gt;
+ *   &lt;module name=&quot;SuppressWarningsHolder&quot;/&gt;
+ * &lt;/module&gt;
+ * &lt;module name=&quot;SuppressWarningsFilter&quot;/&gt;
+ * </pre>
+ * <pre>
+ * class Test {
+ *
+ *     private int K; // violation
```

Comment:
Please copy previous java code block
```
Class Test {
 *   &#64;SuppressWarnings({"membername"}) privat...
```
here
And use same java code again and again.

Reply:
done